tests/test_analysis/test_feature_relevance/test_relevance_table.py

```diff
@@ -34,3 +36,110 @@ def test_model_relevance_table(simple_df_relevance):
     assert np.allclose(relevance_table["regressor_1"]["1"], 1)
     assert np.allclose(relevance_table["regressor_2"]["1"], 0)
     assert relevance_table["regressor_1"]["2"] < relevance_table["regressor_2"]["2"]
+
+
+@pytest.fixture()
+def exog_and_target_dfs():
+    seg = ["a"] * 30 + ["b"] * 30
+    time = list(pd.date_range("2020-01-01", "2021-01-01")[:30])
+    timestamps = time * 2
+    target = np.arange(60)
+    df = pd.DataFrame({"segment": seg, "timestamp": timestamps, "target": target})
+    ts = TSDataset.to_dataset(df)
+
+    cast = ["1.1"] * 10 + ["2"] * 10 + ["56.1"] * 10
```

Comment:
Add nans to this column + test for this case(nans in castable categorical column)